public/scripts/shortcuts.json

```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "answer": "F2",
+    "question": "How do you rename a symbol in VSCode?",
+    "keys": [113],
+    "shortcutType": "mac vscode"
+  },
```

Comment:
Can you move this to the end of the file just so it won't mess up the existing numbering? (and because it's a fairly advanced shortcut compared to the others at the start)

Reply:
Done!